packages/config/config/devices/0x017f/wink_motion_sensor.json

```
@@ -0,0 +1,183 @@
+// Wink Inc. MOTION SENSOR
+// Motion Sensor
+{
+	"manufacturer": "Wink Inc.",
+	"manufacturerId": "0x017f",
+	"label": "Motion Sensor",
+	"description": "Motion Sensor",
+	"devices": [
+		{
+			"productType": "0x0101",
+			"productId": "0x0001",
+			"zwaveAllianceId": 2482
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": {
+		"1": {
+			"label": "Sensitivity Level",
+			"valueSize": 1,
+			"minValue": 8,
+			"maxValue": 255,
+			"defaultValue": 12,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"2": {
+			"label": "Motion Cleared Time Delay",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 5,
+			"maxValue": 600,
+			"defaultValue": 30,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true
+		},
+		"3": {
+			"label": "Basic Set Level",
+			"valueSize": 1,
+			"unit": "%",
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 255,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": true,
+			"options": [
+				{
+					"label": "Turn off",
```

```suggestion
					"label": "Off",
```
Please make these all similar